lectures/aiyagari_jax.md

```diff
@@ -57,6 +57,12 @@ import jax
 import jax.numpy as jnp
 ```
 
+We can also check the `gpu` hardware we are running on using `nvidia-smi`.
```

Comment:
Thanks @mmcky 

How about "Let's check the GPU we are running" 

I'm very keen on cutting words ATM.  Either way `gpu` should be GPU.

Reply:
@jstac done.